src/sql/workbench/browser/designer/designer.ts

```diff
@@ -926,11 +1012,15 @@ export class Designer extends Disposable implements IThemable {
 						}
 					}
 				});
-
+				table.onBlur((e) => {
+					disabledActions.forEach(b => b.enabled = false);
+					table.grid.setSelectedRows([]);
+					table.grid.resetActiveCell();
```

Comment:
what happens if you don't have this?

Reply:
If we don't have this then the active cell is not reset and the Move Buttons don't get enabled once the focus is back into the input box cell

Reply:
sounds good.